### PR TITLE
Feature login button

### PR DESCRIPTION
**Pull request by JamieHo3312**
change login button icon for login word

### CODE REVIEW

File: login/app/scripts/app.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ define(['underscore', 'jquery', 'backbone', 'gitcommit', 'jquery.cookie'], funct
                 this.loginToggle();
             }.bind(this), 500);
         },
+        emptyForm: 0,
```
Comment (Wilson-Su) [inline login/app/scripts/app.js]: Fix the name of emptyForm, it is not correct as what it means

File: login/app/scripts/app.js [hunk 2]
```diff
@@ -83,22 +85,25 @@ define(['underscore', 'jquery', 'backbone', 'gitcommit', 'jquery.cookie'], funct
...
+                iconClazz: iconClazz,
+                loginButton: '登入'
             }));
         },
         loginToggle: function() {
             var username = this.ui.username,
                 password = this.ui.password;
             if (username.val().length > 0 && password.val().length > 0) {
-                this.enableSubmit('icon-ok');
+                this.enableSubmit();
```
Comment (Wilson-Su) [inline login/app/scripts/app.js]: You should pass empty string value here

File: login/app/scripts/app.js [hunk 3]
```diff
@@ -83,22 +85,25 @@ define(['underscore', 'jquery', 'backbone', 'gitcommit', 'jquery.cookie'], funct
         },
         disableSubmit: function(iconClazz) {
             this.ui.submit.attr('disabled', 'disabled').addClass('disabled').html(this.iconTemplate({
-                iconClazz: iconClazz
+                iconClazz: iconClazz,
+                loginButton: this.emptyForm ? "" : '登入'
```
Comment (Wilson-Su) [inline login/app/scripts/app.js]: You should use single quote

**Comment by JamieHo3312** [comment]
Update the feature-login-button branch

**Comment by JamieHo3312** [comment]
Update the feature-login-button branch

**Pull request merged**